onprc_ehr/test/src/org/labkey/test/tests/onprc_ehr/ONPRC_RestrictedIssueTest.java

```diff
@@ -0,0 +1,269 @@
+package org.labkey.test.tests.onprc_ehr;
+
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.junit.experimental.categories.Category;
+import org.labkey.test.BaseWebDriverTest;
+import org.labkey.test.Locator;
+import org.labkey.test.TestTimeoutException;
+import org.labkey.test.categories.CustomModules;
+import org.labkey.test.categories.EHR;
+import org.labkey.test.categories.ONPRC;
+import org.labkey.test.pages.issues.DetailsPage;
+import org.labkey.test.pages.issues.InsertPage;
+import org.labkey.test.pages.issues.UpdatePage;
+import org.labkey.test.util.IssuesHelper;
+import org.labkey.test.util.SqlserverOnlyTest;
+import org.labkey.test.util.TestUser;
+
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+
+@Category({CustomModules.class, EHR.class, ONPRC.class})
```

Comment:
CustomModule.EHR is not needed.

Reply:
`CustomModules.class` should be removed from all of the EHR tests.